packages/mux-video/src/CustomVideoElement.js

```diff
@@ -1,4 +1,4 @@
-import "./polyfills/window";
+import "@mux-elements/polyfills/dist/window";
```

Comment:
this could be moved to mux-video's index.ts file to import a `.ts` file instead

Reply:
I feel pretty strongly about not adding the polyfills far away from where they're actually needed.